evaluation.md

```diff
@@ -1,2 +1,21 @@
 # Evaluation
 
+We started with our syntax because it was different and fun. The only change we
+really made was to allow multi-character state names (we didn't think about that
```

Comment:
Yeah, this is helpful, especially because you could conceivably run out of single-width one char characters (although UTF8 is pretty big...)